pkg/kubelet/container/cache.go

```diff
@@ -120,6 +121,17 @@ func (c *cache) UpdateTime(timestamp time.Time) {
 	}
 }
 
+// GetAll gets all PodStatus in the cache
+func (c *cache) GetAll() []*PodStatus {
```

Comment:
Nit: For readability, it might be helpful to place the `GetAll` function definition under `Get`.